src/core/contract_address/starknet_contract_address.rs

```diff
@@ -158,6 +158,7 @@ impl From<StructContractClass> for CairoArg {
     }
 }
 
+// TODO: Maybe this could be hard-coded (to avoid returning a result)?
```

Comment:
what could be hardcoded? the class hash?

Reply:
This function's return value should be constant, so we may be able to hardcode it to avoid having to bundle compiled cairo programs along with the executable.

Reply:
Wait no, not constant, but given the same input it should always generate the same output. What I meant by hard-coded was the algorithm itself (port it to Rust, instead of all that).